src/libc/internal/aarch64.rs

```diff
@@ -0,0 +1,68 @@
+#[inline(never)]
+#[naked]
+#[no_mangle]
+unsafe extern "C" fn __steed_clone() {
+    // Syscall number is passed in x8, syscall arguments in x0, x1, x2, x3, x4.
+    // The arguments are
+    // (flags: c_ulong,           // x0
+    //  child_stack: *mut c_void, // x1
+    //  ptid: *mut c_int,         // x2
+    //  newtls: c_ulong,          // x3
+    //  ctid: *mut c_int)         // x4
+    //
+    // No registers are clobbered, x0 gets the return value.
+    //
+    // We do not clobber any registers, so we don't need to save any.
+    //
+    // The calling convention passes arguments in registers, from x0 to x6.
+    // (fn_: extern "C" fn(*mut c_void) -> *mut c_void, // x0
+    //  child_stack: *mut c_void,                       // x1
+    //  flags: c_ulong,                                 // x2
+    //  arg: *mut c_void,                               // x3
+    //  ptid: *mut pid_t,                               // x4
+    //  newtls: *mut c_void,                            // x5
+    //  ctid: *mut pid_t)                               // x6
+    //
+    // Both ABIs return the function result in x0.
+    //
+    // This means we need the following moves:
+    // x2 -> x0 // flags
+    // x1 -> x1 // child_stack
+    // x4 -> x2 // ptid
+    // x5 -> x3 // newtls
+    // x6 -> x4 // ctid
+    //
+    // We save `fn_` and `arg` on the child stack.
+
+    asm!("
+        // Align the child stack.
+        and x1,x1,#-16
+
+        // Save `fn_` and `arg` on the child stack.
+        stp x0,x3,[x1,#-16]!
+
+        mov x8,#220 // CLONE
```

Comment:
I don't know how to do that. (This value won't change, it would break a lot of programs and Linux is commited to backward-compatiblity on the syscall layer.)